.github/workflows/release-embedding-sdk.yml

```diff
@@ -92,13 +92,6 @@ jobs:
         run: |
           ./bin/embedding-sdk/release_utils.bash update_package_json_template ${{ inputs.sdk_version }}
 
-      # TODO: Remove this step after the build utils script is merged to master
```

Comment:
This was needed when we hadn't merged the first workflow PR to `master`. Backend `bin/embedding-sdk/release_utils.bash` didn't exist on master. That was why this step was needed, but not anymore.